dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/core/process_definition.py

```diff
@@ -189,6 +190,21 @@ def end_time(self, val) -> None:
         """Set attribute end_time."""
         self._end_time = val
 
+    @property
+    def param_json(self) -> Optional[List[Dict]]:
+        """Return param json base on self.param."""
+        if self.param is None or not self.param:
```

Comment:
Aren't these two judgment conditions the same?

Reply:
It is two behavior, one is for `None` and other for `{}`(empty dict).

Reply:
WDYT, should we combine or separate

Reply:
I think we should combine，beause and `None` and `{}` are empty. maybe we can add a comment.

Reply:
I'm going to add this

Reply:
I add f271f5b to correct it

Reply:
LGTM

Reply:
> LGTM

Thanks for your review